app/server.go

```diff
@@ -44,6 +44,7 @@ func main() {
 
 	pflag.Parse()
 	viper.BindPFlags(pflag.CommandLine)
+	viper.BindPFlag("network.interface", pflag.Lookup("interface"))
```

Comment:
we set defaults [here](https://github.com/mushorg/glutton/blob/a285aea68350086d8a337473fa60fce97d2bffa3/glutton.go#L46)